src/internal_events/throttle.rs

```diff
@@ -13,5 +15,10 @@ impl InternalEvent for ThrottleEventDiscarded {
             "events_discarded_total", 1,
             "key" => self.key,
         );
+
+        emit!(ComponentEventsDropped::<INTENTIONAL> {
+            count: 1,
+            reason: "Rate limit exceeded."
+        })
```

Comment:
In accordance with other internal events, we could add a comment to deprecate the two previous `debug!` and `counter!` invocations here

Reply:
Could you show me a link of other places where we deprecated those? I tried to look at your PRs, but couldn't find any deprecation message. I'm trying to make sure this is done similarly to other cases.